handler/posts.go

```diff
@@ -0,0 +1,237 @@
+package handler
+
+import (
+	"errors"
+	"github.com/Chatted-social/backend/app"
+	j "github.com/Chatted-social/backend/jwt"
+	"github.com/Chatted-social/backend/storage"
+	"github.com/dgrijalva/jwt-go"
+	"github.com/gofiber/fiber/v2"
+	"net/http"
+	"strconv"
+	"strings"
+)
+
+type PostStorage struct {
+	handler
+	Secret []byte
+}
+
+func (s PostStorage) REGISTER(h handler, g fiber.Router) {
+	s.handler = h
+	g.Post("/create", s.Create)
+	g.Patch("/update", s.Update)
+	g.Delete("/delete/:id", s.Delete)
+	g.Get("/post/:id", s.ByID)
+	g.Get("/posts/:ids", s.PostsByIDs)
+	g.Get("/user/posts/:user_id", s.UserPosts)
+}
+
+type CreatePostForm struct {
+	Title string `json:"title" validate:"required,min=3,max=32"`
+	Body  string `json:"body" validate:"required,min=3,max=255"`
+}
+
+type UpdatePatchForm struct {
+	ID    int    `sq:"id" json:"id"`
+	Title string `json:"title" validate:"required,min=3,max=32"`
+	Body  string `json:"body" validate:"required,min=3,max=32"`
+}
+
+func (s PostStorage) Create(c *fiber.Ctx) error {
+	form := CreatePostForm{}
+
+	if err := c.BodyParser(&form); err != nil {
+		return c.Status(http.StatusBadRequest).JSON(app.Err(err.Error()))
+	}
+	if err := Validate(&form); err != nil {
+		return c.Status(http.StatusBadRequest).JSON(app.Err(err.Error()))
+	}
+
+	token := strings.Split(c.Get("Authorization"), "Bearer ")[1]
```

Comment:
Instead of this, u should do `jwt.From(c.Locals("user"))`
JWT middleware (jwtware) stores token in context